src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -136,15 +176,16 @@ export const selectAggregatedEarnOpportunitiesByAssetId = createDeepEqualOutputS
             assets,
           })
 
+          const isActiveOpportunityByFilter =
```

Comment:
The same as above, but for an opportunity, not an AssetId. Note there is no `fiatAmount` check here, since this is used for display purposes too and we obviously want to show empty opportunities